packages/block-library/src/latest-posts/edit.js

```diff
@@ -254,10 +274,11 @@ class LatestPostsEdit extends Component {
 									value: item.name || item.value,
 								} ) )
 							}
-							suggestions={ Object.keys( suggestions ) }
+							suggestions={ Object.keys( categorySuggestions ) }
```

Comment:
same, this is from the other PR. if we change in both places we might end up with conflicts 😁